src/App.js

```diff
@@ -1,13 +1,22 @@
 import React from "react";
-
-import Bookings from "./Bookings";
+import Heading from "./Heading.js";
+import TouristInfoCards from "./TouristInfoCards.js";
+import Bookings from "./Bookings.js";
+import Footer from "./Footer.js";
 import "./App.css";
 
 const App = () => {
+  const footerArray = [
```

Comment:
This works fine; because the links are unlikely to change often, maybe simpler just to put them inside the `Footer` component instead of passing them in as a prop.